api/v1beta1/clusterclass_types.go

```diff
@@ -394,6 +394,9 @@ type VariableSchema struct {
 	OpenAPIV3Schema JSONSchemaProps `json:"openAPIV3Schema"`
 }
 
+// +lifted:source=https://github.com/kubernetes/apiextensions-apiserver/blob/v0.28.1/pkg/apis/apiextensions/v1/types_jsonschema.go#L40
+// +lifted:changed
```

Comment:
What's this for?

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/9239#discussion_r1318362593

Reply:
Added a new comment to ask about the marker format. Let's resolve this one for now